### DISCUSSION TITLE
[Bug] Option `effects` already exists

### DISCUSSION THREAD

**Discussion by kashyapshiv** [cloudinary-community/next-cloudinary#197] [Q&A]
Hi there,

Till yesterday everything was working fine with my next js application, but today I am getting this error everywhere I used the CldImage component.

My code for that is 
`
import { CldImage } from 'next-cloudinary';

<CldImage
          alt="gallery"
          key={item.id}
          src={src}
          height="350"
          width="350"
        />`
![image](https://user-images.githubusercontent.com/105337972/235886839-6dd68991-6e4f-40c4-907a-ac80cf5c5cbf.png)

I see on npm page that latest update is 18h before, is this something to do with that? Please help fixing this.

**Comment by colbyfayock** [comment]
@kashyapshiv thanks for the report - this should now be resolved

https://github.com/colbyfayock/next-cloudinary/pull/199

**Reply by kashyapshiv** [reply]
Okay thanks @colbyfayock , I will check it out.
